registrations/tasks.py

```diff
@@ -474,12 +481,32 @@ def get_data(self):
 
         return data
 
+    def check_already_subscribed(self, identity_id):
+        """
+        Ensures that the mother doesn't have any existing prebirth
+        subscriptions
+        """
+        subscriptions = utils.get_subscriptions(identity_id)
+        for subscription in subscriptions:
+            if (
+                    not subscription['active'] or subscription['completed'] or
+                    subscription['process_status'] not in (0, 1)):
+                pass
```

Comment:
Looks like this needs to be a `continue`, at the moment the check would happen regardless of subscription status.

Reply:
I think it would be a good idea to add a test where this `if` statement is True.